source/Cosmos.System2/Graphics/Mode.cs

```diff
@@ -82,89 +76,13 @@ public int CompareTo(Mode other)
             return 0;
         }
 
-        /// <summary>
```

Comment:
do we want to remove the operator overrides?

Reply:
The overrides aren't removed - I only removed the unneeded XMLDoc comments and reformatted the overrides to fit on a single line. You can see the overrides below the line deletions.

Reply:
o i see them now

Reply:
Is there any reason to do that?

Reply:
Usually, the XMLDoc comments aren't visible for operator overloads (and they should be self-documenting). This is mainly for code clarity, as these operators simply use the `CompareTo` methods.